lib/ansible/modules/packaging/os/apt.py

```diff
@@ -90,14 +90,14 @@
     version_added: "2.1"
   upgrade:
     description:
-      - If yes or safe, performs an aptitude safe-upgrade.
+      - If safe, performs an aptitude safe-upgrade.
       - If full, performs an aptitude full-upgrade.
       - If dist, performs an apt-get dist-upgrade.
       - 'Note: This does not upgrade a specific package, use state=latest for that.'
       - 'Note: Since 2.4, apt-get is used as a fall-back if aptitude is not present.'
     version_added: "1.1"
-    choices: [ dist, full, 'no', safe, 'yes' ]
-    default: 'no'
+    choices: [ '', safe, dist, full ]
```

Comment:
This will be an instant break for anyone using 'yes' or 'no' and should probably be deprecated at the same cadence.